server/game/PropertyTypes/BlankingProperty.js

```diff
@@ -0,0 +1,19 @@
+class BlankingProperty {
+    constructor() {
+        this.blanks = {};
+    }
+
+    addBlankingSource(source, includingTraits = false) {
+        this.blanks[source.uuid] = { source: source, includingTraits: includingTraits };
```

Comment:
I understand how you're using `source` here but is it really necessary? It's only being used for removal tracking, but you could do the same thing by tracking the number of "blank (except traits)" and "blank (including traits)". Removing the specific source isn't important here (though may be in other scenarios), just removing 1 of the matching types of blanks.